### PR TITLE
Added patch to enable check if transaction decorator wqs used

### PR DESCRIPTION
**Pull request by burlachenkoy-sorbet**

### CODE REVIEW

File: dist/assertDecoratorWasUsed.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export declare function patchAssertTransactionalDecoratorWasUsed(): void;
```
Comment (meytals) [inline dist/assertDecoratorWasUsed.d.ts:1]: don't think you need to commit dist
maybe we need to add git ignore?

Comment (burlachenkoy-sorbet) [reply dist/assertDecoratorWasUsed.d.ts:1]: We need, that's the way it works in original fork

Comment (koldunb) [reply dist/assertDecoratorWasUsed.d.ts:1]: @meytals It is normal to have dist for npm fork
@burlachenkoy-sorbet did you` run npm run build`?

Comment (burlachenkoy-sorbet) [reply dist/assertDecoratorWasUsed.d.ts:1]: Yes

Comment (alexandermalinovskyi) [reply dist/assertDecoratorWasUsed.d.ts:1]: If changes under `dist` folder are created after running `npm run build`, we are fine

**Review by koldunb — APPROVED** [review]

**Pull request merged**